include/fiction/algorithms/simulation/sidb/time_to_solution.hpp

```diff
@@ -33,14 +32,16 @@ struct time_to_solution_params
     /**
      * Exhaustive simulation algorithm used to simulate the ground state as reference.
      */
-    exhaustive_sidb_simulation_engine engine = exhaustive_sidb_simulation_engine::QUICKEXACT;
+    exact_sidb_simulation_engine engine = exact_sidb_simulation_engine::QUICKEXACT;
     /**
      * Number of iterations of the heuristic algorithm used to determine the simulation accuracy (`repetitions = 100`
      * means that accuracy is precise to 1%).
      */
     uint64_t repetitions = 100;
     /**
-     * Confidence level.
+     * The confidence level represents the probability that the confidence interval calculated from the simulation
+     * contains the true value. For example, a 95 % (0.95) confidence level means that if the simulation were repeated
+     * many times, approximately 95 out of 100 of the calculated confidence intervals would contain the true value.
```

Comment:
Shouldn't we rather use 99.7% as an example, given that that is the default value?

Reply:
oh, yes, you are right!